### PR TITLE
Expect some errors while collecting stats

### LINKED ISSUE #207: Expect some errors while collecting stats
When disabling an partner, it takes some time until the configuration is removed. During that time, collecting the stats from the server fails, but this is expected

https://sentry.io/akvo-foundation/flow-services/issues/530802167/

```
com.google.appengine.repackaged.com.google.api.client.http.HttpResponseException: 404 Not Found
<!DOCTYPE html>
<html lang=en>
  <meta charset=utf-8>
  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
  <title>Error 404 (Page not found)!!1</title>
  <style>
    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}
  </style>
  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>
  <p><b>404.</b> <ins>That’s an error.</ins>
  <p>The requested URL was not found on this server.  <ins>That’s all we know.</ins>

  File "HttpRequest.java", line 1070, in com.google.appengine.repackaged.com.google.api.client.http/execute
  File "OAuthClient.java", line 64, in com.google.appengine.tools.remoteapi/get
  File "RemoteApiInstaller.java", line 411, in com.google.appengine.tools.remoteapi/getAppIdFromServer
  File "RemoteApiInstaller.java", line 374, in com.google.appengine.tools.remoteapi/loginImpl
  File "RemoteApiInstaller.java", line 335, in com.google.appengine.tools.remoteapi/login
...
(22 additional frame(s) were not displayed)

Error trying to get data for akvoflow-116.appspot.com
```

### PR DESCRIPTION
**Pull request by dlebrero**
Fixes #207

Collecting all errors and reporting sentry error if more than 20% fail

### CODE REVIEW

File: backend/src/akvo/flow_services/stats.clj [hunk 1]
```diff
@@ -68,14 +71,30 @@
...
+  (for [server server-list]
+    (e/if-ok (get-stats server kinds)
+             #(conj (calc-stats kinds %) server))))
+
+(defn too-many-errors? [all-data]
+  (let [errors (vec (filter e/error? all-data))]
+    (cond
+      (empty? errors) {:level :info :message "No errors collecting stats" :data ""}
+
+      (< 0.2
```
Comment (iperdomo) [inline backend/src/akvo/flow_services/stats.clj]: Why this _magic_ number?

File: backend/src/akvo/flow_services/stats.clj [hunk 2]
```diff
@@ -68,14 +71,30 @@
...
 (defn get-all-data [server-list kinds]
-  (for [server server-list
-        :let [stats (get-stats server kinds)]
-        :when stats]
-    (conj (calc-stats kinds stats) server)))
+  (for [server server-list]
+    (e/if-ok (get-stats server kinds)
+             #(conj (calc-stats kinds %) server))))
+
+(defn too-many-errors? [all-data]
```
Comment (iperdomo) [inline backend/src/akvo/flow_services/stats.clj]: `too-many-errors` feels generic enough to belong to the `error` namespace?

**Comment by iperdomo** [comment]
We'll need to catch up on this because i don't understand the intention and implementation decisions on the `error` namespace

File: backend/src/akvo/flow_services/stats.clj (hunk 1 above)
Comment (dlebrero) [reply backend/src/akvo/flow_services/stats.clj]: We expect most of the flow instances to work, or what is the same, little to fail. More than 20% broken seems to me a strong indication that something is broken for real.

**Pull request merged**